Update template for new mandatory service (#7)
render-examples/posthog: README.md

This repo can be used to deploy [PostHog] on Render.

> - It uses the [official PostHog Docker image](https://hub.docker.com/r/posthog/posthog) to create a Render Web Service
> - It creates two [Background Worker](https://render.com/docs/background-workers):
>     1. for Celery (which runs various tasks in the background),
>     2. for the PostHog plugin server (which processes incoming data).
> - It spins up a [Render PostgreSQL Managed Database](https://render.com/docs/databases).
> - It runs a [Redis](https://render.com/docs/deploy-redis) instance for cache and keeping track of jobs.

Reviewer: ```suggestion
- Uses the [official PostHog Docker image](https://hub.docker.com/r/posthog/posthog) to create a Render Web Service
- Creates two [Background Workers](https://render.com/docs/background-workers):
    1. Celery for running background tasks
    2. PostHog Plugin Server
- Creates a [Render PostgreSQL Managed Database](https://render.com/docs/databases)
- Creates a [Redis](https://render.com/docs/deploy-redis) instance for caching and asynchronous jobs
```
